### ISSUE TITLE
Instruction Images

### ISSUE THREAD

**Issue by MartinLoeper** [hiwitool/hiwitool#11]
Where are the images for the instruction page which are referenced from "hiwitool/login?instruction=1"?
They are not inside the develop branch under "hiwitool/resources/images/instruction/*.png" where I expect them to be.

**Comment by trenkle** [comment]
Good point! I forgot to send you them. Ningyuan hide that resources earlier because they contained sensitive information about FZI.
